build_scripts/post_audit_issues.py

```
@@ -0,0 +1,150 @@
+import subprocess
+import sys
+
+LABEL = 'automated-audit'
+_existing_titles: set | None = None
+
+
+def _fetch_existing_titles() -> set:
+    global _existing_titles
+    if _existing_titles is not None:
+        return _existing_titles
+    result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)